sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -993,27 +1049,42 @@ def _wrap_in_df(self):
             preserves_partition_by=partitionings.Arbitrary(),
         ))
 
+  @frame_base.with_docs_from(pd.Series)
   @frame_base.args_to_kwargs(pd.Series)
   @frame_base.populate_defaults(pd.Series)
   @frame_base.maybe_inplace
   def duplicated(self, keep):
+    """Only keep=False and keep="any" are supported, other values of keep make
+    this an order-sensitive operation. Note keep="any" is a Beam-specific
+    option that guarantees only one duplicate will be kept, but unlike "first"
```

Comment:
I'm not sure if there's a convention around double quotes versus backticks for names in these doc strings. I would probably prefer backticks (`first`) to quotes. It seems like this is mostly a matter of [preference](https://stackoverflow.com/questions/22256995/restructuredtext-in-sphinx-and-docstrings-single-vs-double-back-quotes-or-back).

Reply:
Well, based on the reStructuredText documentation, I think I'd use double backticks (``foo``) for arg names and other literals. It's not a big deal. I'm not going to call it out again. But it could be a useful convention. I also took a quick look at the Pandas doc strings, and it seems like they're not very strict or consistent about it. So just a nit.

Reply:
Yeah I was on the fence about this. But I think the argument against backticks for me was really just laziness. 

It makes sense to have a policy to use backticks in the docstrings for anything that's referring to an argument or a python literal (so for string literals we'll use backticks _and_ quotes). That will make it clear in the rendered documentation that these things are references to code, not part of the language. I went ahead and updated all the new docstrings to use double-backticks in this way.

I'm not sure what to do about error messages. I don't think it's worth the trouble to use backticks there since users should always see it rendered as plaintext.